onnxruntime/core/protobuf/onnx.in.proto

```diff
@@ -133,10 +133,10 @@ message AttributeProto {
 
   // The type field MUST be present for this version of the IR.
   // For 0.0.1 versions of the IR, this field was not defined, and
-  // implementations needed to use has_field hueristics to determine
+  // implementations needed to use has_field heuristics to determine
```

Comment:
The files under onnxruntime/core/protobuf are copied from https://github.com/onnx/onnx. Please send your changes there.

Reply:
@snnn Thank you for the review.
Done:  https://github.com/onnx/onnx/pull/2571